libs/utils/src/interceptors/timeout.interceptor.ts

```diff
@@ -34,6 +44,8 @@ export class TimeoutInterceptor implements NestInterceptor {
       timeout(this.timeoutMs),
       catchError((err) => {
         if (err instanceof TimeoutError) {
+          const route = context.switchToHttp().getRequest<Request>().url;
+          this.logger.error(`Request took longer than ${this.timeoutMs}ms to process; throwing timeout error`, route);
```

Comment:
Is route here the gateway route or the the route to the external server?

I was thinking specifically about the external call, and I think we'd need to log the entire external base + path to convey enough info to the reader.

That said, if this is the gateway call that was made, it is likely also good to do

Reply:
Yeah. I'm good with merging this, but I'd say we need some of that additional information to close out #768